src/models/author.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (connection, DataTypes) => {
+  const schema = {
+    author: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique: true,
+      validate: {
+        notNull: {
+          msg: 'No author provided.',
+        },
+        notEmpty: {
+          args: [true],
+          msg: 'The author cannot be empty',
+        },
+      },
+    },
+  };
+  const AuthorModel = connection.define('Author', schema);
+  return AuthorModel;
```

Comment:
No need to create the variable here, you can just return the connection statement directly:
```
  return connection.define('Author', schema);
```
You could do this at the end of the book, genre, and reader models too